frontend/utility/AutoUpdateThread.cpp

```diff
@@ -1,30 +1,22 @@
-#include "../win-update/updater/manifest.hpp"
-#include "update-helpers.hpp"
-#include "shared-update.hpp"
-#include "update-window.hpp"
-#include "remote-text.hpp"
-#include "win-update.hpp"
-#include "obs-app.hpp"
+#include "AutoUpdateThread.hpp"
+#include "ui_OBSUpdate.h"
 
-#include <qt-wrappers.hpp>
-#include <QMessageBox>
+#ifdef _WIN32
+#define WIN32_LEAN_AND_MEAN
+#endif
```

Comment:
I generally prefer `WIN32_LEAN_AND_MEAN` to be defined next to the Windows header include, but have seen this change a few times now. Marking with a review comment.

Reply:
See above.